.github/PULL_REQUEST_TEMPLATE/concise.md

```diff
@@ -0,0 +1,17 @@
+## Relevant Issues
+- [Closes/Related To] Issue #XXX
```

Comment:
Would it be usefully to add the [Closes/Related To] to the full pull request template as well?